ui/src/services/k8s/core.js

```diff
@@ -79,10 +78,12 @@ export async function queryPodInNamespace(namespace, podLabel) {
 
 export async function createNamespacedConfigMap(name, namespace, restProps) {
```

Comment:
In the commit message:

- `create the data object,` => `create the field "data"` (object is a bit ambiguous in this context).
- `we could using the path` => `we can use the path`